### DISCUSSION TITLE
BR-BT-00051-0100

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#612] [Q&A] [answered]
Hi Lab !
I am using sdk 1.8
with notice 14 I got this validation error : 
id : BR-BT-00051-0100
location : /pin:PriorInformationNotice/cac:ProcurementProjectLot[1]/cac:TenderingProcess/cac:EconomicOperatorShortList
failedAssertText : For a given lot, if 'Maximum number of candidates to be invited for the second stage of the procedure' (BT-51) was set, then 'The criteria will be used to select the candidates to be invited for the second stage of the procedure' (BT-40) must be 'yes' for at least one of 'Selection criteria' (BG-702) of lots (forms 16, 17, 18, 20, 21, 22, 23 and 24)
diagnostic : BT-51-Lot
see : field:BT-51-Lot

I saw the assert   "value" : "{ND-SecondStage} ${(BT-40-Lot == TRUE) or not(BT-51-Lot is present)}" , but I think in the error description you should add notice 14

But the problem is that BT-40-Lot is not present in 14.json

Bye
Renato

**Comment by YvesJo** [comment]
Hi,
Thanks for reporting this concern, we are looking into this and will get back to you once clarified.

**Reply by YvesJo** [reply, marked as answer]
Hi,
This will get fixed with SDK 1.11
